src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelStateDictionary.cs

```diff
@@ -55,7 +53,7 @@ public ModelStateDictionary(ModelStateDictionary dictionary)
                 throw new ArgumentNullException(nameof(dictionary));
             }
 
-            _innerDictionary = new CopyOnWriteDictionary<string, ModelState>(
```

Comment:
Discussed in person and agreed to leave the copy constructor in place even though we don't call it.